### PR TITLE
SIAMUQ updates

### PR DESCRIPTION
**Pull request by josephmure**
- typo correction to SIAMUQ 2020 presentation
- new SIAMUQ 2022 presentation

### CODE REVIEW

**Comment by sofianehaddad** [comment]
You should add `siamuq2022` to the `build.yml`  to update continuous integration.
Many missing files in `figures`

**Comment by josephmure** [comment]
Yes, for some reason the PDF files were not included in my commit. Gonna fix that as soon as I can.

File: README.rst [hunk 1]
```diff
@@ -80,3 +80,5 @@ OpenTURNS Presentations
   - `Physical constraints for generative adversial networks (ONERA) <https://github.com/openturns/presentation/blob/master/high-dim-meta-2021/GAN_nuages_OT_ONERA.pdf>`_
   - `High dimension meta-modeling as a preconditioner for aeroacoustic simulation (Airbus) <https://github.com/openturns/presentation/blob/master/high-dim-meta-2021/Metamodelisation_Goldstein_Airbus.pdf>`_
   - `Auto-associative models as a nonlinear PCA for dimension reduction (Phimeca) <https://github.com/openturns/presentation/blob/master/high-dim-meta-2021/modele_autoassociatif_PHIMECA.pdf>`_
+
+- `SIAM-UQ 2022 <https://github.com/openturns/openturns.github.io/blob/master/presentation/master/siamuq-2022-slides-OpenTURNS.pdf>`_
```
Comment (jschueller) [inline README.rst:84]: check the url

Comment (josephmure) [reply README.rst:84]: I don't see what's wrong with the URL: it is exactly the same as for SIAMUQ 2020, except "2020" is replaced by "2022".

Comment (jschueller) [reply README.rst:84]: yes, sorry

**Comment by josephmure** [comment]
@jschueller Apparently there *was* a problem: I forgot to update the build.yml file with the following line:

```
cd siamuq2022 && pdflatex -halt-on-error siamuq-2022-slides-OpenTURNS.tex && pdflatex siamuq-2022-slides-OpenTURNS.tex && cd -
```

Do you want to directly add it or should I open a new PR?

**Comment by jschueller** [comment]
I added it manually but now it fails :[ fixing...

**Comment by josephmure** [comment]
Thanks for the debug!

**Pull request merged**